src/box/lua/upgrade.lua

```diff
@@ -1421,6 +1441,7 @@ local handlers = {
     {version = mkversion(2, 10, 5), func = upgrade_to_2_10_5},
     {version = mkversion(2, 11, 0), func = upgrade_to_2_11_0},
     {version = mkversion(2, 11, 1), func = upgrade_to_2_11_1},
+    {version = mkversion(3, 0, 0), func = upgrade_to_3_0_0},
```

Comment:
You should also add a [downgrade](https://github.com/tarantool/tarantool/blob/1e02e0506f89f01d386fdaaed01ca04c54943de5/src/box/lua/upgrade.lua#L1932) handler.

Reply:
Thanks for noticing, added a handler and tests.